src/main/kotlin/app/revanced/patches/duolingo/unlocksuper/fingerprints/UserSerializationMethodFingerprint.kt

```diff
@@ -0,0 +1,20 @@
+package app.revanced.patches.duolingo.unlocksuper.fingerprints
+
+import app.revanced.patcher.extensions.or
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import com.android.tools.smali.dexlib2.AccessFlags
+import com.android.tools.smali.dexlib2.Opcode
+
+object UserSerializationMethodFingerprint : MethodFingerprint(
+    returnType = "V",
+    accessFlags = AccessFlags.PUBLIC or AccessFlags.CONSTRUCTOR,
+    strings = listOf(
+        "betaStatus",
+        "coachOutfit",
+        "globalAmbassadorStatus",
+    ),
+    opcodes = listOf(
+        Opcode.MOVE_FROM16,
+        Opcode.IPUT_BOOLEAN,
+    ),
```

Comment:
Is this pattern required? Aren't the strings enough to resolve the method?

Reply:
nope, there are 3 more methods with this string pattern